src/winetricks

```diff
@@ -10142,6 +10142,145 @@ load_ie8_kb2936068()
 
 #----------------------------------------------------------------
 
+w_metadata ie8_server2003 dlls \
+    title="Internet Explorer 8 WindowsServer2003" \
+    publisher="Microsoft" \
+    year="?" \
+    media="download" \
+    file1="IE8-WindowsServer2003-x64-ENU.exe" \
+    installed_file1="c:/windows/ie8_main.log"
+
+load_ie8_server2003()
+{
+    # See also bug https://bugs.winehq.org/show_bug.cgi?id=34048
+
+    # Installed dlls (only checked system32 / grep -r "This program cannot be run in DOS mode" *):
+    #
+    # admparse.dll  advpack.dll  advpack.dll.mui  browseui.dll  corpol.dll  dxtmsft.dll  dxtrans.dll
+    # hmmapi.dll  html.iec  icardie.dll  idndl.dll  ie4uinit.exe  ie4uinit.exe.mui  ieakeng.dll
+    # ieaksie.dll  ieakui.dll  ieapfltr.dat  ieapfltr.dll  iedkcs32.dll  iedkcs32.dll.mui  ieframe.dll
+    # ieframe.dll.mui  iepeers.dll  ieproxy.dll  iernonce.dll  iertutil.dll  iesetup.dll  ieudinit.exe
+    # ieui.dll  imgutil.dll  inetcpl.cpl  inseng.dll  jscript.dll  jsproxy.dll  licmgr10.dll
+    # msdbg2.dll  msfeedsbs.dll  msfeeds.dll  msfeedssync.exe  mshta.exe  mshta.exe.mui  mshtml.dll
+    # mshtmled.dll  mshtmler.dll  mshtml.tlb  msls31.dll  msrating.dll  msrating.dll.mui  mstime.dll
+    # nlsdl.dll  normaliz.dll  occache.dll  pngfilt.dll  shdocvw.dll  shlwapi.dll  spmsg.dll
+    # spupdsvc.exe  sqmapi.dll  tdc.ocx  url.dll  urlmon.dll  vbscript.dll  verdcbc.tmp  VGX.dll
+    # w_try_regsvr64()webcheck.dll  wieframe.dll  WinFXDocObj.exe  wininet.dll  xmllite.dll
+    #
+    # + bunch of mui files in en-US
+
+    # Somehow it fails for me when WINEARCH=win32 is set; I`m guessing the installer doesn`t support this (?) (Bad exe)
```

Comment:
If you're not sure, don't guess. Preferably figure out why, but if not, just say "doesn't work on 32-bit".

Reply:
done